docker/open-zaak/README.md

```diff
@@ -0,0 +1,65 @@
+# Open Zaak docker configuration
```

Comment:
openzaak?

Reply:
is een afhankelijkheid van odrc, voor de documenten api